homeassistant/components/reolink/config_flow.py

```diff
@@ -120,7 +120,7 @@ async def async_step_user(self, user_input=None) -> FlowResult:
 
     async def async_obtain_host_settings(
```

Comment:
This doesn't need to be a method anymore now I think. We can make it a standalone function in the config flow module.

Reply:
done